src/grammar.coffee

```diff
@@ -258,13 +258,20 @@ grammar =
     o 'This'
   ]
 
+  # The symbols that can be used to access an object member:
+  # identifiers, string literals, and numbers.
+  AccessorIdentifier: [
+    o 'Identifier'
+    o 'AlphaNumeric'
```

Comment:
A possible solution for the number problem could be to replace this line with:

```coffee
o 'STRING'
o 'INTEGER'
```

Then, in `AlphaNumeric` replace `o 'NUMBER'` with `o 'Number'` and add:

```coffee
Number: [
  o 'NUMBER'
  o 'INTEGER'
]
```

and adjust the `NUMBER` regex in the lexer to match integers separately from other decimal numbers and make `Lexer::numberToken` tag integers as such.